lib/src/utils/product.dart

```diff
@@ -1,17 +1,55 @@
 
 class Product {
-  String id;
-  String price;
-  String title;
-  String currencyCode;
+  String id = "";
+  String priceForAndroid ="";
```

Comment:
Why split this into `priceForAndroid` and `priceForIos`?

Android provides this data as a number (in this case `long`, which can be rounded to a double with 2 decimals):
https://developer.android.com/reference/com/android/billingclient/api/ProductDetails.PricingPhase

Reply:
For android will provide complete 'SkuDetails' which contains of price, priceAmountMicros etc. it can be used either number/ String as required.